onnxruntime/core/framework/tunable.h

```diff
@@ -250,40 +251,42 @@ class TunableOp {
   int FindFastestImpl(const ParamsT* params, const std::vector<Op<ParamsT>>& candidates) {
     ITuningContext* ctx = params->TuningContext();
     auto op_sig = Signature();
-    auto param_sig = params->Signature();
-    LOGS_DEFAULT(VERBOSE) << "FindFastestImpl for " << op_sig << '(' << param_sig << ')';
-    auto min_time = std::numeric_limits<double>::infinity();
+    auto params_sig = params->Signature();
+    LOGS_DEFAULT(VERBOSE) << "finding fastest for " << op_sig << '(' << params_sig << ')';
+    auto min_duration_ms = std::numeric_limits<double>::infinity();
     int id = -1;
 
     constexpr const int max_tuning_iter = 100;
     constexpr const int approx_num_iter = 3;
 
     for (size_t i = 0; i < candidates.size(); i++) {
       auto& candidate = const_cast<Op<ParamsT>&>(candidates[i]);
-      if (!IsSupported(candidate, params)) {
-        LOGS_DEFAULT(VERBOSE) << "FindFastestImpl found unsupported " << op_sig << '(' << param_sig << ") id=" << i;
+      auto status = IsSupported(candidate, params);
+      if (!status.IsOK()) {
+        LOGS_DEFAULT(VERBOSE) << "├──unsupported id=" << i << ", " << op_sig << '(' << params_sig << ")";
+        LOGS_DEFAULT(VERBOSE) << "│  reason: " << status.ErrorMessage();
```

Comment:
```text
finding fastest for onnxruntime::rocm::tunable::blas::internal::GemmTunableOp<__half, ck::tensor_layout::gemm::RowMajor, ck::tensor_layout::gemm::RowMajor>(NN_1_32000_4096)
├──found better instance, new best id=0, old id=-1. 0.305538ms, 100 iters.
├──unsupported id=1, onnxruntime::rocm::tunable::blas::internal::GemmTunableOp<__half, ck::tensor_layout::gemm::RowMajor, ck::tensor_layout::gemm::RowMajor>(NN_1_32000_4096)
│  reason: [hipBLASLt] Solution #0 failed: algo 4503 not supported
├──unsupported id=2, onnxruntime::rocm::tunable::blas::internal::GemmTunableOp<__half, ck::tensor_layout::gemm::RowMajor, ck::tensor_layout::gemm::RowMajor>(NN_1_32000_4096)
│  reason: [hipBLASLt] Solution #1 failed: algo 4504 not supported
├── ...
├──found better instance, new best id=9, old id=0. 0.274621ms, 100 iters.
├──found better instance, new best id=11, old id=9. 0.224896ms, 100 iters.
├──found better instance, new best id=12, old id=11. 0.202376ms, 100 iters.
├──skip slow instance id=42
├──skip slow instance id=44
├──skip slow instance id=71
├──skip slow instance id=78
├── ...
```
Now that we want to avoid repeating the signature for every entry, how about keeping this part in a single line like

```text
finding fastest for onnxruntime::rocm::tunable::blas::internal::GemmTunableOp<__half, ck::tensor_layout::gemm::RowMajor, ck::tensor_layout::gemm::RowMajor>(NN_1_32000_4096)
├──found better instance, new best id=0, old id=-1. 0.305538ms, 100 iters.
├──unsupported id=1, reason: [hipBLASLt] Solution #0 failed: algo 4503 not supported
├──unsupported id=2, reason: [hipBLASLt] Solution #1 failed: algo 4504 not supported
├── ...
├──found better instance, new best id=9, old id=0. 0.274621ms, 100 iters.
├──found better instance, new best id=11, old id=9. 0.224896ms, 100 iters.
├──found better instance, new best id=12, old id=11. 0.202376ms, 100 iters.
├──skip slow instance id=42
├──skip slow instance id=44
├──skip slow instance id=71
├──skip slow instance id=78
├── ...
```
But anyway the current change looks good to me.